esrally/mechanic/provisioner.py

```diff
@@ -165,6 +177,9 @@ def prepare(self, binary):
         # determine after installation because some variables will depend on the install directory
         target_root_path = self.es_installer.es_home_path
         provisioner_vars = self._provisioner_variables()
+        
+        # add java options for telemetry devices
+        provisioner_vars.update({"additional_java_settings" : self.telemetry.instrument_candidate_env(self.es_installer.car, self.es_installer.node_name)})
```

Comment:
We could move this to `prepare()`? Also, a simpler option would be to write it like:

```python
provisioner_vars["additional_java_settings"] = self.telemetry.instrument_candidate_env(self.es_installer.car, self.es_installer.node_name)
```

Reply:
Sorry, not sure what you mean, this snippet is already in prepare()..

Reply:
Should i move it to _provisioner_variables?

Reply:
Sorry, I meant `self._provisioner_variables()`, yes. :)